mlflow/shap.py

```diff
@@ -32,3 +38,70 @@ def _log_matplotlib_figure(fig, out_file, artifact_path=None):
     """
     with _log_artifact_contextmanager(out_file, artifact_path) as tmp_path:
         fig.savefig(tmp_path)
+
+
+def log_explanation(predict_function, features, artifact_path=None):
+    """
+    Log the following items generated by SHAP.
+
+        - base values
+        - shap values
+        - summary bar plot
+
+    :param predict_function: A function to compute the output of a model.
+    :param features: A matrix of features to compute SHAP values with.
+    :param artifact_path: A run-relative artifact path the explanation is saved to. If unspecified,
+                          defaults to "shap".
+
+    :return: A URI of the logged explanation.
+
+    .. code-block:: python
+        :caption: Example
```

Comment:
I'll add this example code as `mlflow/examples/shap/explain.py` in a follow-up PR.

Reply:
+1, could we also include a screenshot of the generated artifacts in the docs? I think you already figured out how to do this in https://github.com/harupy/mlflow/tree/shap-demo :)

Reply:
Got it :) I'll take a new sceenshot (because I have renamed the bar plot) and embed it in the docstring.